README.md

```diff
@@ -59,6 +59,15 @@ Consider the following json
 }
 ```
 
+### Creating an Aerospike Document Client
+
+The Aerospike Document Client is instantiated as follows
+
+``` java
+   AerospikeClient client = new AerospikeClient(AEROSPIKE_SERVER_IP, AEROSPIKE_SERVER_PORT);
```

Comment:
Mention in a comment that there are other constructors for AerospikeClient.

Reply:
There will be only 1 after the next PR (documentBinName redesign).

Reply:
I meant [this](https://docs.aerospike.com/apidocs/java/com/aerospike/client/AerospikeClient.html) one.